test_regress/driver.pl

```diff
@@ -2599,6 +2602,12 @@ =head1 DRIVER ARGUMENTS
 
 =over 4
 
+=item --asan
+
+Enable address sanitizer to compile Verilated C++ code.
+It detects misuse of memory such as out-of-bound access, use-after-free,
+and memory leak.
```

Comment:
Minor fix then feel free to push, thanks!

```suggestion
Enable address sanitizer to compile Verilated C++ code.
This may detect misuses of memory, such as out-of-bound accesses, use-after-free,
and memory leaks.
```